include/goal_game_bonus.h

```diff
@@ -0,0 +1,19 @@
+#ifndef GUARD_GOAL_GAME_BONUS_H
+#define GUARD_GOAL_GAME_BONUS_H
+
+#include "data.h"
+
+struct GoalGameBonus {
+    struct Object2 obj2;
+    u32 unkB4;
+    u32 unkB8;
+    struct Kirby *kirby;
+    u16 unkC0;
+    u16 unkC2;
+    s16 unkC4;
+};
```

Comment:
Can you add the size here in a comment like in other structs? it's useful for quickly recognizing identical structs.

Reply:
Done!